batched/dense/unit_test/CMakeLists.txt

```diff
@@ -74,6 +98,14 @@ IF (KOKKOS_ENABLE_OPENMPTARGET)
   #     backends/Test_OpenMPTarget_Batched_Dense.cpp
   #   COMPONENTS batched_dla
   # )
+
+  # KOKKOSKERNELS_ADD_UNIT_TEST(
+  #   batched_gemm_openmptarget
+  #   SOURCES
+  #     ${PACKAGE_SOURCE_DIR}/test_common/Test_Main.cpp
+  #     backends/Test_OpenMPTarget_Batched_GEMM.cpp
+  #   COMPONENTS batched_dla
+  # )
```

Comment:
Can these be removed?

Reply:
No we really want to keep openmptarget as we will try again to build these tests when compiler and linker make progress.